src/olympia/devhub/api_views.py

```diff
@@ -0,0 +1,28 @@
+from olympia import amo
+from olympia.addons.views import AddonVersionViewSet
+from olympia.api.permissions import (
+    AllowAddonAuthor, AllowReviewer, AllowReviewerUnlisted, AnyOf)
+from olympia.devhub.models import ActivityLog
+from olympia.devhub.serializers import ActivityLogSerializer
+from olympia.versions.models import Version
+
+
+class VersionReviewNotesViewSet(AddonVersionViewSet):
```

Comment:
(That means you'll need to move `get_addon_object()` to a mixin, but that's fine)